addons/mail/data/mail_data.xml

```diff
@@ -402,5 +402,21 @@
 <p></p>
 </div>
         </template>
+        <!-- Admin technical notifications template -->
+        <template id="mail_channel_notify_admins">
+            <div>
+                <p><b><t t-esc='subject'/></b></p>
+                <t t-raw='message'/><br/><br/>
```

Comment:
Yes actually this was used because html parsing needed for message format but to prevent t-raw issues I removed html from messages and set views urls in a separate container (see ffba891 & 4b2d277)